app/presenters/finder_presenter.rb

```diff
@@ -278,6 +278,12 @@ def all_content_finder?
     self.content_item['content_id'] == 'dd395436-9b40-41f3-8157-740a453ac972'
   end
 
+  # FIXME: This should be removed once we have a way to determine
+  # whether to display metadata in the finder definition
+  def eu_exit_finder
+    slug == "/find-eu-exit-guidance-business"
```

Comment:
Would it not be better to use the `content_id` instead of the slug?  This will also match how `all_content_finder?` works. Could I also suggest adding `?` to the end of the method name to follow ruby conventions?

Reply:
If we going to make this method public could we also write a test for it?